File: client1/message_handler.py
import requests
import threading

import constants

class MessageHandler:
    def send(self, data):
        threading.Thread(target=self.createRequest, args=(data)).start()
        # print(request.text)

    def createRequest(self, data):
        return requests.post(constants.END_POINT_ADDRESS, json=data)

File: generator2/Generator2/MessageHandler.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Net.Http;
using System.Net;
using System.IO;
using Generator2.Model;

namespace Generator2
{
    class MessageHandler
    {
        public void Send(string data)
        {
            var httpWebRequest = (HttpWebRequest)WebRequest.Create(Constants.END_POINT_ADDRESS);
            httpWebRequest.ContentType = "application/json";
            httpWebRequest.Method = "POST";

            using (var streamWriter = new StreamWriter(httpWebRequest.GetRequestStream()))
            {
                streamWriter.Write(data);
            }

            httpWebRequest.GetResponse();
        }
    }
}
